deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -371,6 +371,18 @@ extension WebxdcViewController: WKScriptMessageHandler {
 extension WebxdcViewController: WKURLSchemeHandler {
     func webView(_ webView: WKWebView, start urlSchemeTask: WKURLSchemeTask) {
         if let url = urlSchemeTask.request.url, let scheme = url.scheme, scheme == INTERNALSCHEMA {
+            if url.path == "/webxdc-update.json" || url.path == "webxdc-update.json" {
```

Comment:
are both conditions needed or was this just for testing?

Reply:
the first is needed, the second might be (I don't know maybe other web view versions), I kept it for redundancy. we could remove it but I think it won't hurt